build/prefix-source-map.jsm

```diff
@@ -17,4 +17,5 @@
 
 this.EXPORTED_SYMBOLS = [ "SourceMapConsumer", "SourceMapGenerator", "SourceNode" ];
 
+Components.utils.import("resource://gre/modules/devtools/Console.jsm");
```

Comment:
Yeah left it in on purpose so we don't have to add it again when debugging which is annoying.